analytics_data_api/v0/serializers.py

```diff
@@ -332,6 +332,14 @@ class LearnerSerializer(serializers.Serializer, DefaultIfNoneMixin):
     enrollment_date = serializers.DateField(source='enrollment_date', format=settings.DATE_FORMAT)
     cohort = serializers.CharField(source='cohort')
 
+    def transform_segments(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
+
+    def transform_cohort(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
```

Comment:
I can definitely add in a check for an empty string cohort.  Why not?

Out of curiosity, is it a valid cohort name?  It's an odd use case to support, especially when filtering.